srcpkgs/gpsd/template

```diff
@@ -105,6 +105,7 @@ gpsd-devel_package() {
 		vmove usr/include
 		vmove "usr/lib/pkgconfig/libgps*.pc"
 		vmove usr/lib/libgps.so
```

Comment:
Could these be combined to be just `vmove "usr/lib/*.so"`?

Reply:
This whole section could be changed to

    vmove usr/include
    vmove usr/lib/pkgconfig
    vmove "usr/lib/*.so"
    vmove usr/share/man/man3

Reply:
in such scenarios, implicit subpackages (like alpine does) seem appealing

Reply:
Ah, I didn't think about the other `-devel` package.  This is fine as is